dll/win32/advpack/files.c

```diff
@@ -381,7 +383,9 @@ static HRESULT DELNODE_recurse_dirtree(LPWSTR fname, DWORD flags)
         {
 #ifdef __REACTOS__
 deleteinitialdirectory:
+#endif
             TRACE("%s: directory\n", debugstr_w(fname));
+#ifdef __REACTOS__
```

Comment:
```suggestion
            TRACE("%s: directory\n", debugstr_w(fname));
#ifdef __REACTOS__
deleteinitialdirectory:
```
(This is a suggestion similar to what's already done in the next case for a file.)

Reply:
That would skip the `TRACE()`, in the `deleteinitialdirectory` case.
Do you confirm you want to change current behaviour?

Reply:
Ah yes, good catch @SergeGautherie . Let's keep it as it currently is in your PR.